integrationtest/performance/orm/src/test/java/org/hibernate/search/test/performance/TestRunnerArquillian.java

```diff
@@ -100,7 +102,7 @@ private static Asset createPersistenceXml() {
 
 	public static Asset manifest() {
 		String manifest = Descriptors.create( ManifestDescriptor.class )
-				.attribute( "Dependencies", "org.apache.commons.lang" )
+				.attribute( "Dependencies", "org.apache.commons.lang,com.google.guava" )
```

Comment:
Where is the Guava requirement coming from? Can we avoid it?

Reply:
The Stopwatch class. The one from Apache only has millisecond resolution. If that's okay with you, then we can switch to the Apache one; otherwise we'll have to code our own, and I don't see the point as this is only a test.

Reply:
ok, let's use the Guava one.

Reply:
BTW I don't see how it can be more precise that milliseconds :) but ok that's not too relevant.

Reply:
It uses `System.nanoTime()`. But yeah, maybe it's about time drift rather than resolution, I keep confusing those things.